workloads/kube-burner/common.sh

```diff
@@ -11,7 +11,8 @@ else
   export PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
   export METADATA_COLLECTION=${METADATA_COLLECTION:-true}
 fi
-OPERATOR_REPO=https://github.com/cloud-bulldozer/benchmark-operator.git
+operator_repo=${OPERATOR_REPO:=https://github.com/cloud-bulldozer/benchmark-operator.git}
```

Comment:
We might want to document the vars/defaults - OPERATOR_REPO and OPERATOR_BRANCH @mohit-sheth.